cogs/groupsCog.py

```diff
@@ -51,7 +51,24 @@ async def glist(self, ctx,
     async def gdestroy(self, ctx, group_prefix: str = commands.parameter(description="Your group (short name)")):
         group_name = f"{group_prefix}_{ctx.author}"
         status, result_str = organize_group.destroy_group(group_name)
-        # if exit_status, delete channels and role
+        # if status, delete channels and role
+        status=True
+        if status:
+            # remove role
+            role = discord.utils.get(ctx.guild.roles, name=group_name)
+            if role:
+                await role.delete()
+
+            # remove category
+            category = discord.utils.get(ctx.guild.categories, name=group_name)
+            if category:
```

Comment:
It is helpful for server logs to also log deletion of category.

Reply:
What do you mean by that? Do you mean an info log on that? All entered commands are already logged. What would you log additionally?